### ISSUE TITLE
Training done, but always return error code -6

### ISSUE THREAD

**Issue by stereomatchingkiss** [NVIDIA/DIGITS#1366]
I am using digits detect net to train a fish detector but it always return error code -6 after training done(100% finished).

I follow the instructions on [this page](https://devblogs.nvidia.com/parallelforall/deep-learning-object-detection-digits/), use google net as pretrained network, set the parameters as the page mentioned and copy the detectnet_network.prototxt on [here](https://raw.githubusercontent.com/NVIDIA/caffe/caffe-0.15/examples/kitti/detectnet_network.prototxt). I do not changed the source(source: "examples/kitti/kitti_train_images.lmdb") of the network but select the dataset created by digits as the page mentioned.

Format of the label looks like

"fish 1.0 0 0.0 555 433 786 636 0.0 0.0 0.0 0.0 0.0 0.0 0.0"

```
Train net output #0: loss_bbox = 0 (* 2 = 0 loss)
Train net output #1: loss_coverage = 0.281627 (* 1 = 0.281627 loss)
Iteration 609, lr = 6.48465e-08
Iteration 616 (4.42979 iter/s, 1.58021s/7 iter), loss = 0.494803
Train net output #0: loss_bbox = 0 (* 2 = 0 loss)
Train net output #1: loss_coverage = 0.494803 (* 1 = 0.494803 loss)
Iteration 616, lr = 6.11978e-08
Snapshotting to binary proto file snapshot_iter_620.caffemodel
Snapshotting solver state to binary proto file snapshot_iter_620.solverstate
Iteration 620, Testing net (#0)
Ignoring source layer train_data
Ignoring source layer train_label
Ignoring source layer train_transform
Test net output #0: loss_bbox = 0 (* 2 = 0 loss)
Test net output #1: loss_coverage = 0.214023 (* 1 = 0.214023 loss)
Test net output #2: mAP = 0
Test net output #3: precision = 0
Test net output #4: recall = 0
Optimization Done.
Optimization Done.
```
[train_val.prototxt](http://pastebin.com/eM47Gftw)
[solver.prototxt](http://pastebin.com/Ri5SubUw)
[original.prototxt](http://pastebin.com/irwJHmjM)
[caffe_output.log](http://pastebin.com/y4AHXpn7)

Any idea what is happening?Thanks

**Comment by lukeyeager** [comment]
1. Is this an actual problem or just a UI issue? Can you still use the trained models or not?
2. Does this happen for other models? Try running through the [LeNet on MNIST example](https://github.com/NVIDIA/DIGITS/blob/digits-5.0/docs/GettingStarted.md).
3. How did you install Caffe?
4. What are the versions of your OS, CUDA Driver, CUDA Toolkit and cuDNN library?

**Comment by lukeyeager** [comment]
Hmm, that all looks fine to me - no red flags.
